src/lib/operators/aggregate_hash.cpp

```diff
@@ -297,7 +297,8 @@ void AggregateHash::_aggregate() {
    */
   for (ColumnID column_id{0}; column_id < _aggregates.size(); ++column_id) {
     const auto& aggregate = _aggregates[column_id];
-    if (!aggregate.column && aggregate.function == AggregateFunction::Count) {
+    if (aggregate.column == INVALID_COLUMN_ID) {
+      Assert(aggregate.function == AggregateFunction::Count, "Only COUNT(*) may (and must) have an invalid ColumnID.");
```

Comment:
It becomes `COUNT(*)` by virtue of having an invalid ColumnID, so this feel a bit circular. Suggestion: "Only COUNT may have an invalid ColumnID". The fact that we are talking about `COUNT(*)` becomes clear in the next line